### ISSUE TITLE
Use x-splade-data within x-splade-form

### ISSUE THREAD

**Issue by JapSeyz** [protonemedia/laravel-splade#242]
- Laravel Version: 9.47.0
- PHP Version: 8.1.14
- Splade JS Version (npm): 1.1.5
- Splade PHP Version (composer): 1.1.5

You may use the [built-in Artisan Command](https://splade.dev/docs/upgrading) to show the installed versions of the packages:

```bash
php artisan splade:show-versions
```

### Description:

Is it possible to use `x-splade-data` within a form?

The idea is to remember the message that the user saved, in localstorage. My use case allowing users to contact each other, but the message is gonna be almost the same a lot of the time, so it makes sense to provide their latest message to them.

Example:

```blade
<x-splade-data remember="contact-message" local-storage>
    <x-splade-textarea name="message" v-model="data.message" autosize/>
</x-splade-data>
```

The problem with this, is that once I wrap it inside the `x-splade-data` then it's no longer seen by the `x-splade-form` that is surrounding the `x-splade-data`, thus the message is saved to localstorage, but no longer submitted in the form.

**Comment by codersuraz** [comment]
You are using `data.message`, but if you want that to work within the form, you must use `form.message` on v-model.

Wrap your form within x-splade-data and do 
```
<x-splade-form default={message: data.message}>
 <x-splade-textarea name="message" autosize/>
</x-splade-form>
```

And if you are using a splade form component, you don't need to use `v-model`. Just `name` attribute is enough.

**Comment by iMusicJJ** [comment]
Unfortunately, this doesn't seem to save the new state in the local-storage. It may pull from there, but I have a hard time testing that, as I can't get it to save anything to there in the first place.

**Comment by pascalbaljet** [comment]
I've got a working example, but one line of code must be updated in the Data component 😅

I'll tag a new release soon and post the example!

**Comment by pascalbaljet** [comment]
I've included the change in v1.2.6:

```blade
<x-splade-data remember="some-key" local-storage>
    <x-splade-form default="{ message: data.message }">
        <input
            @input="data.message = form.message"
            v-model="form.message"
        />

        {{-- or --}}

        <x-splade-input
            @input="data.message = form.message"
            name="message"
        />

    </x-splade-form>
</x-splade-data>
```

Make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).

**Comment by iMusicJJ** [comment]
It works. Thanks!
